internal/host/refresh_status_preprocessor.go

```diff
@@ -54,6 +55,12 @@ func (r *refreshPreprocessor) preprocess(c *validationContext) (map[string]bool,
 		st := v.condition(c)
 		stateMachineInput[v.id.String()] = st == ValidationSuccess
 		message := v.formatter(c, st)
+
+		// skip the fallowing validations when installation is done
+		if v.id == IsConnected && c.host.Progress.CurrentStage == models.HostStageDone {
```

Comment:
is this enough? i think we should validate it when host got to reboot state. From this point there is no reason to validate if it is connected